modules/@apostrophecms/schema/ui/apos/components/AposInputWrapper.vue

```diff
@@ -190,6 +192,10 @@ export default {
   }
 }
 
+.apos-field__label-meta:empty {
+  display: none;
+}
```

Comment:
Removes meta span container if it's empty (avoid to screwing up display with new range info part)